Sming/Libraries/ModbusMaster/samples/generic/app/application.cpp

```diff
@@ -47,12 +46,12 @@ void mbLoop()
 void preTransmission()
 {
 	digitalWrite(RS485_RE_PIN, HIGH);
-	delayMilliseconds(2);
+	//delayMilliseconds(1);
 }
 
 void postTransmission()
 {
-	delayMicroseconds(500);
+	//delayMicroseconds(500);
```

Comment:
Remove the commented line too.

Reply:
Should I remove also 
```
		/*
		for(i = 0; i < nrOfRegisters; i++) {
			buffer[i] = mbMaster.getResponseBuffer(i);
			debugComPort.printf("Reg %d: %d\r\n", i, buffer[i]);
		}
		*/
```
then?

Reply:
The pasted and commented code is not part of your current changes. If this if-block is replaced by something else or is not useful at all then you can also remove it.

Reply:
I'll leave it. Improved example should take into account the number of registers to read/write. Maybe I'll do it later.